modules/node/src/transfer/transfer.service.ts

```diff
@@ -201,16 +202,47 @@ export class TransferService {
       throw new Error(`App with provided hash has not been installed: ${linkedHash}`);
     }
 
+    const freeBalanceAddr = this.cfCoreService.cfCore.freeBalanceAddress;
+
     const freeBal = await this.cfCoreService.getFreeBalance(
```

Comment:
Just realized we should make helper functions at the channel service level to wrap `getNodeFreeBalance`.